suites/rados/singleton/all/ec-lost-unfound-upgrade.yaml

```diff
@@ -0,0 +1,25 @@
+roles:
+- - mon.a
+  - mon.b
+  - mon.c
+  - osd.0
+  - osd.1
+  - osd.2
+- - osd.3
+tasks:
+- install:
+    branch: infernalis
```

Comment:
@dachary This commit introduces tests that install infernalis and then upgrade, so I guess the commit should be dropped from this PR. Agree?